common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1089,20 +1088,20 @@ def has_ended(self):
         Returns True if the current time is after the specified course end date.
         Returns False if there is no end date specified.
         """
-        if self.end is None:
-            return False
-
-        return datetime.now(UTC()) > self.end
+        return course_metadata_utils.has_course_ended(self.end)
```

Comment:
Update docstring. I believe had_course_ended returns the string if true, else returns None if  false.